tools/build_file_generator/third_party_deps/create_third_party_deps_zip.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```

Comment:
Why did we rewrite the java code in bash?
Couldn't we run the java code on the same input as the bash code?